src/app/@ansyn/ansyn/app-providers/overlay-source-providers/open-aerial-source-provider.ts

```diff
@@ -37,7 +37,7 @@ export class OpenAerialSourceProvider extends BaseOverlaySourceProvider {
 		let bbox = bboxFromGeoJson(fetchParams.region as GeoJSON.Polygon);
```

Comment:
1. what about time ? (fetchParams.time -> start, end) , it's critical.
2. Replace `%2C` with simple `,`